src/sass/common/reality-check.scss

```diff
@@ -21,7 +21,7 @@
     background-color: #fff;
     border-radius:    .3em;
 }
-#reality-check > div > div > div:first-child {
+.lightbox > div > div > div:first-child {
```

Comment:
Thanks, I added id to both the div in reality-check & balances. Recognise straight by these id is more straight forward.